core/amber/src/main/scala/edu/uci/ics/texera/web/TexeraWebApplication.scala

```diff
@@ -127,14 +129,22 @@ class TexeraWebApplication extends io.dropwizard.Application[TexeraWebConfigurat
     bootstrap.addBundle(new WebsocketBundle(classOf[CollaborationResource]))
     // register scala module to dropwizard default object mapper
     bootstrap.getObjectMapper.registerModule(DefaultScalaModule)
-
-    if (AmberConfig.sinkStorageMode.equalsIgnoreCase("mongodb")) {
+    if (AmberConfig.isUserSystemEnabled) {
```

Comment:
the old config is on `mongodb`, which may not be equivalent to user sys?

Reply:
The old config is not correct anymore. We want to clean up both Mongo and log files periodically. The log file will be created if we have MySQL. Also, this change can cover the case if we decide to use another storage to store the results.